src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/ChainPal.cs

```diff
@@ -49,7 +41,7 @@ internal sealed partial class ChainPal : IDisposable, IChainPal
             return status.dwError == 0;
         }
 
-        public IEnumerable<X509ChainElement> ChainElements
+        public X509ChainElement[] ChainElements
```

Comment:
This isn't introduced by your change @justinvp, but @bartonjs, I'd feel better if property getters didn't introduce array allocations like this on every call... makes it too easy to accidentally introduce significant performance problems, e.g. by doing an iteration like ```for (int i = 0; i < cp.ChainElements.Length; i++) { var element = cp.ChainElements[i]; ... }```.  Should we change this to be a method?  I've not looked at the rest of the type, but there might be some other similar properties.

That's just a note for later and shouldn't prevent this PR from going ahead.